src/components/user/DeleteButton.tsx

```diff
@@ -76,9 +76,19 @@ const DeleteButton: FunctionComponent = () => {
     return <ConfirmDialog
       prompt='Delete your account?'
```

Comment:
Are these out of scope for this PR? `ConfirmDialog` seems to also have some strings, that are untranslated.

Reply:
On my backlog of things to do is clean up the deletion confirmation, was unsure if we'll want these strings to stick around so thought I'd leave them for now, but can localize now if preferable.

Reply:
No, if they are likely to change, let's keep them out of scope.